C/README.md

```diff
@@ -1 +1,3 @@
-## Add the links to files you have added over here while creating a PR.
\ No newline at end of file
+## Add the links to files you have added over here while creating a PR.
+
+AP program in C-https://github.com/vanisha23/DS-Algo-Point/blob/AP/C/AP.c
```

Comment:
Write like this in README.md file

`
-[Find nth term in A.P.](AP.c)
`

Make this change.